client/my-sites/upgrades/domain-search/domain-search.jsx

```diff
@@ -165,9 +166,12 @@ var DomainSearch = React.createClass( {
 
 module.exports = connect(
 	function( state, props ) {
+		const selectedSite = props.sites.getSelectedSite();
```

Comment:
I'd suggest that we don't introduce another instance of `sitesList.getSelectedSite()` - we're trying to get rid of them, see #8726. How about using the `getSelectedSite()` selector instead:

```
import { getSelectedSite } from 'state/ui/selectors';
...
const selectedSite = getSelectedSite( state );
```